### ISSUE TITLE
Improve alt text for animations on the static site homepage

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4031]
## Area or Component
Static site homepage - simplereport.gov

## Quick Summary
The alt text for the two animations on the homepage just reads "animation 1" and "animation 2", which is not descriptive or helpful. These alt texts need to be update to more accurately describe the images.

## Accessibility Violation
Short text alternative is not appropriate: https://cdc-devtools.dequecloud.com/rule-help/en/alt-text-short-text-not-meaningful

## Action Requested
Add better alternative text for the two animations.

## Screenshots (optional)
Animation 1 (on the right):
![Screen Shot 2022-07-21 at 5 29 27 PM](https://user-images.githubusercontent.com/80282552/180336814-e0ae9c4d-9d1a-4b93-9162-258da610c7c4.png)

Animation 2 (on the left):
![Screen Shot 2022-07-21 at 5 29 17 PM](https://user-images.githubusercontent.com/80282552/180336811-7113a5d5-6549-4682-aae9-9fb4cadcd0c0.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=856291185&range=17:18
